datasetinsights/estimators/faster_rcnn.py

```diff
@@ -110,6 +114,7 @@ def __init__(
 
         if checkpoint_file:
             self.checkpointer.load(self, checkpoint_file)
+        self.mltracking = TrackerFactory.create(config, const.MLFLOW_TRACKER)
```

Comment:
I will suggest using explicit name such as `mlflow_tracking`. It's not clear to readers when `mltracking` is used, as kubeflow and tensorboard are both mltracking stuff.

Reply:
also, for consistency, use tracker instead of tracking here